atlasmap-maven-plugin/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>io.atlasmap</groupId>
     <artifactId>atlas-parent</artifactId>
-    <version>2.5.x-SNAPSHOT</version>
+    <version>3.0.0</version>
```

Comment:
This needs to be 3.0.0-SNAPSHOT. The release process automatically changes to the non-SNAPSHOT version number